akka-docs/src/main/paradox/typed/index.md

```diff
@@ -22,5 +22,5 @@ project.description: Using Akka to build reliable multi-core applications distri
 * [coexisting](coexisting.md)
 * [style-guide](style-guide.md)
 * [from-classic](from-classic.md)
-
+* [lombok](immutable.md)
```

Comment:
I don't think it is important enough to go as a direct toc entry under "actors", perhaps we could put it under "project information" instead, where we already have "IDE tips" which is somewhat (although vaguely) related?